node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -1036,11 +1036,11 @@ impl Initialized {
 				ctx.send_message(ChainSelectionMessage::RevertBlocks(blocks_to_revert.clone()))
 					.await;
 			} else {
-				gum::error!(
+				gum::trace!(
 					target: LOG_TARGET,
 					?candidate_hash,
 					?session,
-					"Could not find parent block info for concluded candidate!"
+					"Could not find an including block for candidate against which a dispute has concluded."
```

Comment:
nit: Could this indicate a bug or unusual condition? I would have this higher than `trace`, maybe even `warn`?

Reply:
I'm really not sure what level of logging is appropriate here. It seems that under normal conditions we could see a concluded dispute for which we haven't seen an including block yet. In that case we'd rely on the slower reversion triggering from on-chain revert logs I think.